### PR TITLE
refactor(rust): remove shared lookup.json file and make it local to nodes

### PR DESCRIPTION
**Pull request by adrianbenavides**
Now each node has its own copy of the lookup.json file. No more shared files!

Drawback: everytime a node is created (embedded or background), if the identity has been enrolled and therefore, it has access to the cloud resources, it will fetch the available spaces/projects to fill the lookup data. I have plans to improve this, but for now I just wanted to make things work without introducing new stuff.

Context: https://github.com/build-trust/codebase/issues/402

Orchestrator pipeline ✅ : https://github.com/build-trust/ockam-artifacts/actions/runs/4002593030

### CODE REVIEW

**Comment by adrianbenavides** [comment]
@mergifyio rebase

File: implementations/rust/ockam/ockam_api/src/nodes/service/portals.rs [hunk 1]
```diff
@@ -168,7 +168,8 @@ impl NodeManagerWorker {
                 .and_then(|p| {
                     if let Some(p) = p.cast::<Project>() {
                         node_manager
-                            .projects
+                            .projects()
+                            .unwrap()
```
Comment (caoakleyii) [inline implementations/rust/ockam/ockam_api/src/nodes/service/portals.rs]: We should handle this gracefully, instead of panicing.

Comment (adrianbenavides) [reply implementations/rust/ockam/ockam_api/src/nodes/service/portals.rs]: Fixed https://github.com/build-trust/ockam/pull/4075/files#diff-5b3f34a98f0adf81c262908c1b6ee444f6e61e4242075b18860c9a57ea986e3fR168-R170

**Comment by adrianbenavides** [comment]
Closed in favor of https://github.com/build-trust/ockam/pull/4192

**Pull request closed without merging**